src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -545,7 +558,7 @@ private void OnAsyncTaskStateChanged(DynamoScheduler sender, TaskStateChangedEve
         /// Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.
         /// </summary>
         /// <filterpriority>2</filterpriority>
-        public void Dispose()
+        public virtual void Dispose()
```

Comment:
There were changes to RevitDynamoModel, then they got backed out. Thanks for noticing this Ben.